docs/src/iterate.md

```diff
@@ -20,7 +20,7 @@ julia> itp = interpolate(ones(3,3), BSpline(Linear()));
 julia> kiter = knots(itp);
 
 julia> collect(kiter)
-3×3 Array{Tuple{Int64,Int64},2}:
+3×3 $(Array{Tuple{Int,Int}, 2}):
```

Comment:
The string interpolation only works on docstrings and not on markdowns:

```julia
"""
$(...)
"""
```

The documentation is built against Julia 1.6; I'm not sure of it, maybe https://juliadocs.github.io/Documenter.jl/dev/man/doctests/#Fixing-Outdated-Doctests could help.

Reply:
Ahh, and here I thought that would be a one and done ☹️